modules/dists/BlockDist.chpl

```diff
@@ -1568,6 +1613,60 @@ proc BlockArr.doiCanBulkTransferStride() param {
   return useBulkTransferDist;
 }
 
+proc BlockArr.oneDData {
```

Comment:
Good idea on the quick-check -- I'll add that.
Thought about computing this at setup time.  At the time I was thinking it might be more coding, but now that I've seen the version I did write I'm realizing I might be wrong about that.  For now I'll probably just modify the TODOs to note this alternative.